### DISCUSSION TITLE
how to unregister an autocmd?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [neoclide/coc.nvim#5054] [Q&A] [answered]
We have workspace.registerAutocmd().

However, how to unregister an autocmd?

**Comment by fannheyward** [comment]
No such feature yet. But I can't get your point about to unregister an autocmd, the `workspace.registerAutocmd` is used for extension to register an autocmd at needed, when you want to unregister one?

**Reply by Freed-Wu** [reply]
The requirement does exist: https://github.com/black-desk/fcitx5-ui.nvim/blob/master/lua/fcitx5-ui/init.lua#L148-L149

**Reply by Freed-Wu** [reply, marked as answer]
Currently, I have to

```js
workspace.nvim.command('au! coc_dynamic_autocmd InsertCharPre');
```
